lib/rsa.js

```diff
@@ -1735,10 +1739,10 @@ function _getMillerRabinTests(bits) {
  * @return true if detected, false if not.
  */
 function _detectSubtleCrypto(fn) {
-  return (typeof window !== 'undefined' &&
-    typeof window.crypto === 'object' &&
-    typeof window.crypto.subtle === 'object' &&
-    typeof window.crypto.subtle[fn] === 'function');
+  return (typeof util.getGlobalScope() !== 'undefined' &&
```

Comment:
Micro optimization would be to do lift out `var globalScope = util.getGlobalScope()` above to avoid repeated calls.